files/puppet_helper.groovy

```diff
@@ -180,6 +186,108 @@ class Util {
   }
 } // class Util
 
+/**
+ * Helper class to be used when creating a new instance of FileCredentialsImpl
+ */
+public class DirectFileItem implements FileItem {
```

Comment:
I'm not familiar with this interface.  I'm guessing that using [DiskFileItem](https://commons.apache.org/proper/commons-fileupload/apidocs/org/apache/commons/fileupload/disk/DiskFileItem.html) directly is a PITA?